jobs/src/main/scala/dcos/metronome/model/CronSpec.scala

```diff
@@ -1,55 +1,37 @@
 package dcos.metronome.model
 
-import com.cronutils.model.definition.{ CronDefinition, CronDefinitionBuilder }
-import com.cronutils.model.time.ExecutionTime
-import com.cronutils.model.Cron
-import com.cronutils.parser.CronParser
+import it.sauronsoftware.cron4j.Predictor
 import org.joda.time.DateTime
 
 import scala.util.control.NonFatal
 
-class CronSpec(val cron: Cron) {
+class CronSpec(val cron: String) {
 
-  private[this] lazy val executionTime: ExecutionTime = ExecutionTime.forCron(cron)
+  new Predictor(cron)
```

Comment:
required to valid cron when cronSpec is initialized.

Reply:
Should it be moved to the corresponding object instead, `CronSpec` is changed to accept `Predictor`, not `String`? Just wondering if it makes sense.